### DISCUSSION TITLE
Datahub GraphQL Update a product image

### DISCUSSION THREAD

**Discussion by noelhunter** [pimcore/pimcore#10959] [Q&A / Support] [answered]
Can someone provide an example of how to update an image type field on a product?  Here is what I am trying:

`mutation {  
    updateCar(
        id: 190, 
        input: {    
            name: "Datahub-car-2",
            categories: [
                {id: 1132, type: "object"}
            ],
            image: [
                {
                id: 18
                }
            ]
            }
        ) 
    {
        success message output {
            id
            name
        }
    }
}`

I get an error: Field \"updateCar\" argument \"input\" requires type ImageInput, found [{id: 18}]."

**Comment by cutec-chris** [comment, marked as answer]
with 

image: { id: 18 } 

this seems to work

**Reply by noelhunter** [reply]
For reference, here is a sample update that worked for me:

`mutation {  
    updateCar(
        id: 5240, 
        input: {    
            name: "Sample",
            categories: [
                {id: 1132, type: "object"}
            ],
            image:  {id: 1731}
            
        } 
        ) 
    {
        success message output {
            id
            name
        }
    }
}`
